server/src/auth_service.rs

```diff
@@ -102,11 +105,40 @@ async fn get_refresh<Backend>(
     data: web::Data<AppState<Backend>>,
     request: HttpRequest,
 ) -> TcpResult<HttpResponse>
+where
```

Comment:
Why do we need to update the refresh token route? We don't give out a refresh token, I don't expect it to even work.